src/Controls/src/Core/Platform/Android/TabbedPageManager.cs

```diff
@@ -655,76 +655,70 @@ protected virtual ColorStateList GetItemTextColorStates()
 
 		protected virtual ColorStateList GetItemIconTintColorState()
 		{
-			if (IsBottomTabPlacement)
-			{
-				if (_orignalTabIconColors is null)
-					_orignalTabIconColors = _bottomNavigationView.ItemIconTintList;
-			}
-			// this ensures that existing behavior doesn't change
-			else if (!IsBottomTabPlacement && BarSelectedItemColor != null && BarItemColor == null)
-				return null;
+			if (_orignalTabIconColors is null)
+				_orignalTabIconColors = IsBottomTabPlacement ? _bottomNavigationView.ItemIconTintList : _tabLayout.TabIconTint;
 
 			Color barItemColor = BarItemColor;
 			Color barSelectedItemColor = BarSelectedItemColor;
 
-			if (barItemColor == null && barSelectedItemColor == null)
+			if (barItemColor is null && barSelectedItemColor is null)
 				return _orignalTabIconColors;
 
-			if (_newTabIconColors != null)
+			if (_newTabIconColors is not null)
 				return _newTabIconColors;
 
 			int defaultColor;
+			int checkedColor;
 
 			if (barItemColor is not null)
-			{
 				defaultColor = barItemColor.ToPlatform().ToArgb();
-			}
 			else
-			{
+				defaultColor = GetDefaultColor();
```

Comment:
I would keep the brackets, we prefer it.

Reply:
Changed